product_docs/docs/pgd/4/rel_notes/pgd_4.3.5_rel_notes.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.5"
+navTitle: "Version 4.3.5"
+redirects:
+  - /pgd/latest/bdr/release_notes/bdr4.3.5_rel_notes/
+---
+
+Released: 16 May 2024
+
+EDB Postgres Distributed version 4.3.5 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.10, 15.5, and later.
+!!!
+
+| Component | Version | Type        | Description                                                                                                    | Addresses      |
```

Comment:
Because the column is for support tickets and CVEs. And also because "Related Support Tickets" is very wide and would push the actual description into around 25 characters wide. I can change it to "Ticket/CVE" which isn't pretty but would denote what the column is.

Reply:
Changed to "Ticket"